solarnet/playbooks/domains.yaml

```diff
@@ -0,0 +1,108 @@
+---
+- hosts: jenkins
+  vars:
+      do:
+          DO_API_KEY: "{{ lookup('env','DIGITAL_OCEAN_API_KEY') }}"
+          DO_CLIENT_ID: "{{  lookup('env','DIGITAL_OCEAN_CLIENT_ID') }}"
+  tasks:
+    - name: "install dopy"
+      pip: name=dopy
+    - environment: do
+      digital_ocean_domain:
+        state: present
+        name: uranus.nyc.srv.protocol-dev.com
```

Comment:
we should be able to manage the `i.ipfs.io` domain from DO. i'll make sure you have creds.